bin/wordpress-packager

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env php
+<?php
+declare(strict_types=1);
+
+require_once(dirname(__DIR__) . '/vendor/autoload.php');
+
+use CzProject\GitPhp\Git;
+use Roots\WordPressPackager\License;
+use Roots\WordPressPackager\Package\Package;
+use Roots\WordPressPackager\Package\Writer;
+use Roots\WordPressPackager\ReleaseSources\Concerns\ReleaseType;
+use Roots\WordPressPackager\ReleaseSources\SourceInterface;
+use Roots\WordPressPackager\Target;
+use Roots\WordPressPackager\Util\Directory;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\SingleCommandApplication;
+use Symfony\Component\Filesystem\Filesystem;
+
+(new SingleCommandApplication())
+    ->setName('WordPress Packager')
+    ->addArgument('remote', InputArgument::REQUIRED, 'Git remote repository')
+    ->addArgument('package', InputArgument::REQUIRED, 'Package name')
+    ->addOption('source', null, InputOption::VALUE_REQUIRED, 'Release source', 'WPDotOrgAPI')
+    ->addOption('type', 't', InputOption::VALUE_REQUIRED, 'Release type', ReleaseType::Full->value)
+    ->addOption('instable', 'i', InputOption::VALUE_NONE, 'Includes instable')
```

Comment:
instable should be "unstable"? couple other references throughout

Reply:
Indeed, just rebased on my working branch where this was fixed 🙂